frappe/core/doctype/server_script/server_script.json

```diff
@@ -49,14 +49,15 @@
    "in_list_view": 1,
    "in_standard_filter": 1,
    "label": "Reference Document Type",
-   "options": "DocType"
+   "options": "DocType",
+   "search_index": 1
   },
   {
    "depends_on": "eval:doc.script_type==='DocType Event'",
    "fieldname": "doctype_event",
    "fieldtype": "Select",
    "label": "DocType Event",
-   "options": "Before Insert\nBefore Validate\nBefore Save\nAfter Insert\nAfter Save\nBefore Submit\nAfter Submit\nBefore Cancel\nAfter Cancel\nBefore Delete\nAfter Delete\nBefore Save (Submitted Document)\nAfter Save (Submitted Document)\nOn Payment Authorization\nOn Payment Paid\nOn Payment Failed"
+   "options": "Before Insert\nBefore Validate\nBefore Save\nAfter Insert\nAfter Save\nBefore Rename\nAfter Rename\nBefore Submit\nAfter Submit\nBefore Cancel\nAfter Cancel\nBefore Delete\nAfter Delete\nBefore Save (Submitted Document)\nAfter Save (Submitted Document)\nOn Payment Authorization\nOn Payment Paid\nOn Payment Failed"
```

Comment:
can you add tests for this? see test_server_script.py

Reply:
Yeah, I'm working on test cases. 😃